example/ruby/sample.rb

```diff
@@ -0,0 +1,5 @@
+require 'openssl'
+require './github'
+
```

Comment:
hmm, I think for this short sample it's easier to understand by embedding github.rb into here. How do you think?